ts/BUILD.bazel

```diff
@@ -1,10 +1,33 @@
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@bazel_skylib//rules:common_settings.bzl", "int_flag")
 
 exports_files(
     glob(["*.bzl"]),
     visibility = ["//docs:__pkg__"],
 )
 
+# Enable with --@aspect_rules_ts//ts:skip_lib_check=1
+int_flag(
```

Comment:
I'd use a string flag here. `true` `false` `unspecified` or something close to this. self-describing values are better.

Reply:
and limit it via `values` attribute to prevent user errors. https://github.com/bazelbuild/bazel-skylib/blob/141432789c92e9db2402ef0be58e2a2d2c4dd1fd/rules/common_settings.bzl#L92

Reply:
yeah, OTOH I did that for stamping once in rules_docker and it was a mistake  because -1,0,1 had already won in many repos, and it seemed better to be consistent with trinary flags.

Reply:
Maybe two Boolean flags is even better. Not sure if we need error handling in case you set both as they are meant to be mutually exclusive

Reply:
now changed to `string_flag`